gammapy/estimators/points/core.py

```diff
@@ -387,6 +387,12 @@ def to_table(self, sed_type=None, format="gadf-sed", formatted=False):
                 tables.append(table_flat)
 
             table = vstack(tables)
+
+            table.meta["TIMESYS"] = time_axis.reference_time.scale
```

Comment:
Was there any reason why we could not use https://github.com/gammapy/gammapy/blob/main/gammapy/utils/time.py#L99?

Reply:
Or even better we could introduce `TimeMapAxis.to_table()`, with the given format...

Reply:
For now, I have used the `time_ref_to_dict` function.